2017/WG-09-12.md

```diff
@@ -0,0 +1,71 @@
+![WebAssembly logo](/images/WebAssembly.png)
+
+## Agenda for the September 12th video call of WebAssembly's Working Group
+
+- **Host**: Google Hangouts
+- **Dates**: Tuesday September 12th, 2017
+- **Times**: 9:00am–10:00am Pacific Time
+- **Location**: *same Google Hangouts as previous CG video meetings*
+- **Contact**:
+    - Name: Brad Nelson
+    - Email: bradnelson@google.com
+
+### Registration
+
+None required if you've attended before. Email Brad Nelson to sign up if it's
+your first time. The meeting is open to WG members only (we'll likely broaden
+this in future meetings).
+
+## Logistics
+
+The meeting will be a Google Hangouts call.
+
+## Agenda items
+
+1. Opening, welcome and roll call
+    1. Opening of the meeting
+    1. Introduction of attendees
+1. Find volunteers for note taking.
+1. Adoption of the agenda
+1. Proposals and discussions
+    1. Openness (Brad Nelson)
+       1. Discussion on how open the WG process should be.
```

Comment:
Can you link to W3C guidance on WG openness here? I'd like to understand what's up for discussion versus what's set in stone.

Reply:
Asking for some more on this.
We have the option for instance to keep the meetings closed. I don't think we want to do that, in fact I'd rather it be wide open, but I wanted to have that be an explicit decision.

Reply:
What do you mean "asking for more info"? I think this section should link to the specific W3C docs.

Reply:
Sorry meant asking Eric.
In any event, have added links and rephrased the poll slightly. It sounds like the main constraint is that we need to attest that artifact produced by the WG come from folks bound by the proper agreements.